src/tests/databases/ncbi_db/table.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE `ncbi_taxa_name` (
+  `taxon_id` int(10) unsigned NOT NULL,
+  `name` varchar(255) NOT NULL,
+  `name_class` varchar(50) NOT NULL,
+  KEY `taxon_id` (`taxon_id`),
+  KEY `name` (`name`),
+  KEY `name_class` (`name_class`)
+) ENGINE=MyISAM DEFAULT CHARSET=latin1;
+
+CREATE TABLE `ncbi_taxa_node` (
+  `taxon_id` int(10) unsigned NOT NULL,
+  `parent_id` int(10) unsigned NOT NULL,
+  `rank` char(32) NOT NULL DEFAULT '',
+  `genbank_hidden_flag` tinyint(1) NOT NULL DEFAULT '0',
+  `left_index` int(10) NOT NULL DEFAULT '0',
+  `right_index` int(10) NOT NULL DEFAULT '0',
+  `root_id` int(10) NOT NULL DEFAULT '1',
+  PRIMARY KEY (`taxon_id`),
+  KEY `parent_id` (`parent_id`),
+  KEY `rank` (`rank`),
+  KEY `left_index` (`left_index`),
+  KEY `right_index` (`right_index`)
+) ENGINE=MyISAM DEFAULT CHARSET=latin1;
```

Comment:
SQLite version, since automated tests are run with SQLlite (or you'll have to override the travis.yml context for this test)
```suggestion
create table ncbi_taxa_name
(
    taxon_id   bigint       not null,
    name       varchar(255) not null,
    name_class varchar(50)  not null
);

create index ncbi_taxa_name__index_1 on ncbi_taxa_name (taxon_id);
create index ncbi_taxa_name__index_2 on ncbi_taxa_name (name);
create index ncbi_taxa_name__index_3 on ncbi_taxa_name (name_class);

create table ncbi_taxa_node
(
    taxon_id            bigint   not null,
    parent_id           bigint   not null,
    rank                char(32) not null default '',
    genbank_hidden_flag tinyint  not null default '0',
    left_index          int      not null default '0',
    right_index         int      not null default '0',
    root_id             int      not null default '1',
    primary key (taxon_id)
);
create index ncbi_taxa_node__index_1 on ncbi_taxa_node (parent_id);
create index ncbi_taxa_node__index_2 on ncbi_taxa_node (rank);
create index ncbi_taxa_node__index_3 on ncbi_taxa_node (left_index);
create index ncbi_taxa_node__index_4 on ncbi_taxa_node (right_index);
```